ci/build_windows.sh

```diff
@@ -11,8 +12,13 @@ cd build
 #  https://github.com/actions/virtual-environments/issues/687
 #  https://github.com/actions/virtual-environments/blob/master/images/win/Windows2019-Readme.md
 
+# print version
+cmake --version
+
 cmake \
   -G "Visual Studio 16 2019" -A x64 \
+  -DVCPKG_TARGET_TRIPLET="${VCPKG_DEFAULT_TRIPLET}" \
+  -DCMAKE_BUILD_TYPE=Release \
```

Comment:
Isn't this set with `--config Release` below? Why does this need to be defined twice?

Reply:
I was confused by this as well, but they mean different things. Providing it as part of the cmake call tells CMake to generate a specific Makefile (or otherwise platform build system) configuration and exclude others. Providing it as an argument to --build tells CMake to only build that configuration. i.e. `cmake -DCMAKE_BUILD_TYPE=Release` `cmake --build --config Debug` would fail. The Windows Debug builds still need improving.